srcpkgs/lolcat-c/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'lolcat-c'
+pkgname=lolcat-c
+version=0.0.2017.12.20
+revision=1
+_commit=e298c102bd51ae67fbae09df5dfc00cba16a2231
```

Comment:
https://github.com/jaseg/lolcat/commit/e298c102bd51ae67fbae09df5dfc00cba16a2231
This commit is dated 20 March 2017.
Your package should be `version=0.0.0.2017.03.20`